<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interw</class>
 <widget class="QMainWindow" name="interw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="temprature">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>841</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Gestion de livraison</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>781</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">tabWidget_2 = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tab_livr">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>291</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_recherche">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>30</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_livr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>adresse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_recherche">
        <property name="geometry">
         <rect>
          <x>642</x>
          <y>30</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>70</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;ID_livr&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>340</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="colone_tri">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>70</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_livr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>adresse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="imprimer">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="adresse">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>20</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>import</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dt">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>290</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>791</width>
          <height>541</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:tranparent;

border-image:url(:/image/1.jpg);

background:none;

border:none;

background-repeat:none;
Aa</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="adrlivraison">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ffffff;&quot;&gt;date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="datelivraison">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter_2">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>150</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="excel">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>330</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>excel</string>
        </property>
       </widget>
       <widget class="QLabel" name="clock">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="date">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <zorder>label_2</zorder>
       <zorder>tab_livr</zorder>
       <zorder>cb_recherche</zorder>
       <zorder>le_recherche</zorder>
       <zorder>supprimer</zorder>
       <zorder>label_21</zorder>
       <zorder>id_3</zorder>
       <zorder>label_37</zorder>
       <zorder>label_38</zorder>
       <zorder>modifier_2</zorder>
       <zorder>colone_tri</zorder>
       <zorder>imprimer</zorder>
       <zorder>PDF</zorder>
       <zorder>adresse</zorder>
       <zorder>commandLinkButton</zorder>
       <zorder>pushButton_5</zorder>
       <zorder>dt</zorder>
       <zorder>adrlivraison</zorder>
       <zorder>label_5</zorder>
       <zorder>label_7</zorder>
       <zorder>datelivraison</zorder>
       <zorder>Ajouter_2</zorder>
       <zorder>excel</zorder>
       <zorder>clock</zorder>
       <zorder>date</zorder>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>550</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des livreurs</string>
     </attribute>
     <widget class="QTabWidget" name="affichage2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>771</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tab_liv_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>331</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_recherche_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>50</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_livreur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom_liv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prenom_liv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>num_tel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>salaire</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="rech">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>50</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QComboBox" name="colone_tri_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_livreur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom_liv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prenom_liv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>num_tel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>salaire</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="supprimer_2">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>80</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="imprimer_2">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PDF_2">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>230</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_39">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ffffff;&quot;&gt;id&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>47</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_41">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_42">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;numtel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>230</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>290</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="numtel_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>320</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>260</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="salaire_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>350</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ffffff;&quot;&gt;salaire&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton_2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>40</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>771</width>
          <height>481</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:tranparent;

border-image:url(:/image/1.jpg);

background:none;

border:none;

background-repeat:none;
Aa

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;nom_liv&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;prenom_liv&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;num_tel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numtel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffffff;&quot;&gt;salaire&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="salaire">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter1">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="excell">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>excel</string>
        </property>
       </widget>
       <widget class="QLabel" name="img">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>380</y>
          <width>161</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="upload">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>440</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>upload image</string>
        </property>
       </widget>
       <widget class="QLabel" name="date_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="clock_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <zorder>label_6</zorder>
       <zorder>tab_liv_2</zorder>
       <zorder>cb_recherche_2</zorder>
       <zorder>rech</zorder>
       <zorder>colone_tri_2</zorder>
       <zorder>supprimer_2</zorder>
       <zorder>imprimer_2</zorder>
       <zorder>PDF_2</zorder>
       <zorder>label_39</zorder>
       <zorder>label_40</zorder>
       <zorder>label_41</zorder>
       <zorder>label_42</zorder>
       <zorder>id_2</zorder>
       <zorder>prenom_2</zorder>
       <zorder>numtel_3</zorder>
       <zorder>modifier3</zorder>
       <zorder>nom_2</zorder>
       <zorder>salaire_2</zorder>
       <zorder>label_43</zorder>
       <zorder>commandLinkButton_2</zorder>
       <zorder>nom</zorder>
       <zorder>label_16</zorder>
       <zorder>label_20</zorder>
       <zorder>prenom</zorder>
       <zorder>label_22</zorder>
       <zorder>numtel</zorder>
       <zorder>label_23</zorder>
       <zorder>salaire</zorder>
       <zorder>ajouter1</zorder>
       <zorder>excell</zorder>
       <zorder>img</zorder>
       <zorder>upload</zorder>
       <zorder>date_2</zorder>
       <zorder>clock_2</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
